src/molecule_podman/driver.py

```diff
@@ -29,6 +29,10 @@
 from molecule.util import lru_cache
 
 log = logger.get_logger(__name__)
+# To change the podman executable, set environment variable
+# MOLECULE_PODMAN_EXECUTABLE
+# An example could be MOLECULE_PODMAN_EXECUTABLE=podman-remote
+podman_exec = os.environ.get("MOLECULE_PODMAN_EXECUTABLE", "podman")
```

Comment:
This should probably be done in some configuration layer rather than on a module-level at import-time.

Reply:
I've moved it from the module-level to the __init__ function, I hope this is better? I'm unfortunatly pretty new at using python in this way, so plese let me know, if there is a better way to do it :)